benchalerts/tests/unit_tests/mocked_responses/GET_conbench_benchmarks_run_id_contender_wo_base.json

```diff
@@ -0,0 +1,114 @@
+{
+    "status_code": 200,
+    "data": [
+        {
+            "batch_id": "some-batch-uuid-1",
```

Comment:
For future: need a bit of a plan to update mocked responses once we update API behavior / do integration testing with less mocking.

Reply:
Absolutely!! I'm hoping we can rely a lot less on brittle mocks once we do #720 some more.